IOSSecuritySuite/IOSSecuritySuite.swift

```diff
@@ -139,38 +139,23 @@ public extension IOSSecuritySuite {
     Usage example
     ```
     C:
-     void denyDebugger() {
-     }
-     let amIMSHookFunction = amIMSHookFunction(denyDebugger) ? true : false
+        void denyDebugger() {
+        }
+        let amIMSHookFunction = amIMSHookFunction(denyDebugger) ? true : false
 
     Swift:
      1.
+        func denyDebugger() {
+        }
+     
         typealias functionType = @convention(thin) ()->()
     
         func getSwiftFunctionAddr(_ function: @escaping functionType) -> UnsafeMutableRawPointer {
             return unsafeBitCast(function, to: UnsafeMutableRawPointer.self)
         }
-     
-        func denyDebugger() {
-        }
     
         let func_addr = getSwiftFunctionAddr(denyDebugger)
         let amIMSHookFunction = amIMSHookFunction(func_addr) ? true : false
-     
-     2.
```

Comment:
it is not correct to get swift's vtable function pointer after I tested it.
so, I delete this use case